src/getTime.js

```diff
@@ -18,7 +18,9 @@
  * @returns {string} - valid time
  */
 function getTime(str) {
-  // write code here
+  const validTime = str.match(/\b([0-1]\d|2[0-3]):[0-5]\d\b/);
+  const firstValidTime = validTime === null ? '' : validTime.join().slice(0, 5);
```

Comment:
you can write simple
```suggestion
  return validTime ?  validTime[0] : '';
```